inc/class-gutenberg-ramp-criteria.php

```diff
@@ -74,6 +80,55 @@ public function set( $criteria ) {
 		return false;
 	}
 
+	/**
+	 * Merge all criteria from all calls to ramp_for_gutenberg_load_gutenberg()
+	 *
+	 * @param array $criteria The new criteria to be merged with existing criteria.
+	 * @return array The merged criteria.
+	 */
+	public function merge( $criteria = [] ) {
+
+		// Get our existing criteria
+		$existing_criteria = self::$criteria;
+
+		// If this hasn't been set to active yet, there's nothing to merge - theoretically this should never be triggered.
+		$ramp = Gutenberg_Ramp::get_instance();
```

Comment:
Makes sense. I think I've lost a bit of my bearings after moving everything around so I'm not sure exactly _why_ this was in there.

Do we really need to check anything here now, though? `$criteria` shouldn't be empty at this point, and `merge()` is only being called if `$existing_criteria` is not `null`. So, if the function is being called, it's because both `$criteria` and `$existing_criteria` have something to merge.

Reply:
I think it's always good to make sure the variables contain the data you're expecting, so for example - checking for empty and/or is_array() does make sense.

Reply:
> merge() is only being called if $existing_criteria

Reply:
Good point. Not trusting input should also include not trusting users to use your functions the way you intended. :)